src/marine/hgodas_insitu2ioda.py

```diff
@@ -65,9 +70,7 @@ def _read(self):
         qcs = ncd.variables['qc'][:]
         ncd.close()
 
-        base_date = datetime(1970, 1, 1) + timedelta(seconds=int(time[0]))
-
-        self.VarAttrs['depth', 'MetaData']['units'] = 'm'
+        dt = qcs.astype('int64')
```

Comment:
I thought I commented on this line before, or perhaps it was in another of the 3 files.  I believe this line is a possible mistake.  The variable ```qcs``` should clearly be filled with QC values and nothing to do with time and I thought the variable called ```dt``` was something related to delta-time.  Please have a closer look at this.

Reply:
The purpose for this sentence is to define an array dt, and it is used in below:

**dt[i]** = time[0] + hrs[i]*3600
locKey = lats[i], lons[i], dpth[i], **dt[i]**

It is a correct sentence.

Reply:
Ah, I see now that dt is simply being created as a copy of another variable to gain its length and then reassigned later.  Kinda kludgy and I would personally prefer to create an empty and zero filled array using something like this:
```dt = np.full(len(qcs), 0, dtype=np.int64)```

however I will allow the existing solution.

Reply:
I second @gthompsnJCSDA's suggestion. The code is more clear about what is the intention (create a new variable with a matching length to qcs).

Reply:
Now I used waterZonalVelocity and  waterMeridionalVelocity as Greg suggested.